addons/sale_timesheet/models/project_update.py

```diff
@@ -19,46 +19,50 @@ def _get_template_values(self, project):
 
     @api.model
     def _get_profitability_values(self, project):
-        if not (self.user_has_groups('project.group_project_manager') and (project.analytic_account_id and project.allow_billable or project.allow_timesheets)):
+        costs_revenues = project.analytic_account_id and project.allow_billable
+        timesheets = project.allow_timesheets and self.user_has_groups('hr_timesheet.group_hr_timesheet_user')
+        if not (self.user_has_groups('project.group_project_manager') and (costs_revenues or timesheets)):
             return {}
-        profitability = project.get_profitability_common()
-        start_of_month = fields.Date.context_today(self) + relativedelta(day=1)
-        timesheets_this_month = self.env['project.profitability.report'].read_group(
-            [('project_id', '=', project.id),
-             ('line_date', '>=', start_of_month)],
-            ['project_id',
-             'timesheet_unit_amount'],
-            ['project_id'])
-        timesheets_previous_month = self.env['project.profitability.report'].read_group(
-            [('project_id', '=', project.id),
-             ('line_date', '>=', start_of_month + relativedelta(months=-1, day=1)),
-             ('line_date', '<', start_of_month)
-             ],
-            ['project_id',
-             'timesheet_unit_amount'],
-            ['project_id'])
-        timesheet_unit_amount = timesheets_this_month and timesheets_this_month[0]['timesheet_unit_amount'] or 0.0
-        previous_timesheet_unit_amount = timesheets_previous_month and timesheets_previous_month[0]['timesheet_unit_amount'] or 0.0
-        return {
-            'allow_timesheets': project.allow_timesheets,
+        this_month = fields.Date.context_today(self) + relativedelta(day=1)
+        previous_month = this_month + relativedelta(months=-1, day=1)
+        result = {
+            'allow_timesheets': timesheets,
+            'allow_costs_and_revenues': costs_revenues,
             'analytic_account_id': project.analytic_account_id,
-            'month': start_of_month.strftime('%B %Y'),
-            'previous_month': (start_of_month + relativedelta(months=-1, day=1)).strftime('%B'),
+            'month': _('%(month_year)s', month_year=this_month.strftime('%B %Y')),
+            'previous_month': _('%(month)s', month=previous_month.strftime('%B')),
```

Comment:
I am really unsure that this will get translated by transifex.
Couldn't we use format_date?
If not, maybe we could use something like this in the template?:
```xml
<t t-esc="moment(profitability['variable_name']).format('MMM YYYY')"/>
```

Reply:
You're right, I'll try and use the format_date :)

NB : The template is used server-side so it doesn't have access to any JS library (like `moment`)